IFC4x3/Properties/s/SourceArea_1BMVyQRuL1zRS08BFGzp4P/Documentation.md

```diff
@@ -0,0 +1 @@
+Description of the area at the origin of the water inflow.
```

Comment:
Description? Maybe `IfcText` would be more suitable then?

Reply:
Fixed in [a9b58f3](https://github.com/bSI-InfraRoom/IFC-Specification/pull/602/commits/a9b58f34288b501e0a4e15fcb5c1c01a067b9cf4)